src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using `box-sizing: border-box;` on individual elements. Instead, you can use a global selector to apply it to all elements for consistency across the project.